lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,26 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use Text::Trim;
+use DDG::Spice;
+
+attribution github => ['https://github.com/iambibhas', 'Bibhas'],
```

Comment:
Just noticed this wasn't finished yet! I was looking for a sample query now that it's deployed...